src/Orchard.Web/Modules/Orchard.ContentTypes/Views/Admin/Edit.cshtml

```diff
@@ -2,7 +2,7 @@
 @{
     Style.Require("ContentTypesAdmin");
     Script.Require("jQuery");
-    Layout.Title = T("Edit Content Type - {0}", Model.DisplayName).ToString();
+    Layout.Title = T("Edit Content Type - {0}", Model.DisplayName);

```

Comment:
Displayed title is fixed, but page head's title tag is corrupted : ContentType's DisplayName is encoded twice.